### DISCUSSION TITLE
About how to use the JMX to monitor Trino

### DISCUSSION THREAD

**Discussion by YuanYunjing** [trinodb/trino#13868] [General]
I'm integrating Trino with Alluxio. 

To know why there is no effect shows, I need to monitor the access delay of Hive Metastore. 
But on the trino website, I didn't find the introduction of the JMX Metrics of Trino. Seeing them from SQL query is too hard to find the right Metric to use，so I want to ask if there are some guide to use the JMX Metrics of Trino?

Thank you!

**Comment by findepi** [comment]
https://trino.io/docs/current/admin/jmx.html shows some metrics, but doesn't include any metastore-related metrics
cc @mosabua @jhlodin

**Reply by mosabua** [reply]
That page explains how to see all available metrics with the JMX connector for example. I am not aware of any specific metrics for the metastore access that are useful but I also have not looked in detail. Maybe @findepi knows or someone else can let us know. The page does not try to explain all metrics since there are a lot of them and they change quite a bit.
